src/Components/Map.jsx

```diff
@@ -14,27 +16,55 @@ class Map extends Component {
   };
 
   state = {
-    posts: []
+    posts: [],
+    refreshData: false
   }
 
   componentDidMount() {
     axios.get('/api/v1/posts').then(response => {
       console.log(response)
-      this.setState({ posts: response.data });
+      this.setState({ posts: response.data, refreshData: false });
     });
   }
 
-  render() {
+  componentDidUpdate(prevState) {
+    if (prevState.refreshData !== this.state.refreshData) {
+      axios.get('/api/v1/posts').then(response => {
+        console.log(response)
```

Comment:
No need to keep the console logs when you submit PR for final review.

Reply:
Yes, removed